README.md

```diff
@@ -19,4 +19,135 @@ Grafana | [grafana](https://github.com/open-cluster-management/grafana) | Grafan
 Dashboard Loader | [grafana-dashboard-loader](https://github.com/open-cluster-management/grafana-dashboard-loader) | Sidecar proxy to load grafana dashboards from configmaps. 
 Management Ingress | [management-ingress](https://github.com/open-cluster-management/management-ingress) | NGINX based ingress controller to serve Open Cluster Management services. 
 Observatorium API | [observatorium](https://github.com/open-cluster-management/observatorium) | API Gateway which controls reading, writing of the Observability data to the backend infrastructure. Forked from main observatorium API repo.
-Thanos Ecosystem | [kube-thanos](https://github.com/open-cluster-management/kube-thanos) | Kubernetes specific configuration for deploying Thanos. The observatorium operator leverages this configuration to deploy the backend Thanos components.
\ No newline at end of file
+Thanos Ecosystem | [kube-thanos](https://github.com/open-cluster-management/kube-thanos) | Kubernetes specific configuration for deploying Thanos. The observatorium operator leverages this configuration to deploy the backend Thanos components.
+
+## Quick Start Guide
+
+### Prerequisites
+
+- go version v1.15+.
+- docker version 17.03+.
+- kubectl version v1.16.3+.
+- Access to a Kubernetes v1.11.3+ cluster.
+
+Use the following quick start commands for building and testing the Multicluster Observability Operator:
+
+### Clone the Repository
+
+Check out the multicluster-observability-operator repository.
+
+```
+$ git clone git@github.com:open-cluster-management/multicluster-observability-operator.git
+$ cd multicluster-observability-operator
+```
+
+### Build the Operator
+
+Build the multicluster-observability-operator image and push it to a public registry, such as Quay.io.
+
+```
+$ make -f Makefile.prow build
+$ docker build -f Dockerfile -t quay.io/<YOUR_USERNAME_IN_QUAY>/multicluster-observability-operator:latest .
+$ docker push quay.io/<YOUR_USERNAME_IN_QUAY>/multicluster-observability-operator:latest
+```
+
+### Run the Operator in the Cluster
+
+1. Before you deploy the Multicluster Observability Operator in the cluster, you need to installed the following dependencies:
+
+- The [cert-manager](https://github.com/open-cluster-management/cert-manager) is deployed into the cluster
+- The following required CRDs are installed in the cluster:
```

Comment:
remove the crds dependency by adding logic to check if placementrule exists or not in placementrule controller.